### ISSUE TITLE
Direct Streams no longer appear to work. Get an Automated Request error from google

### ISSUE THREAD

**Issue by clandestine8** [l3uddz/drive_strm#18]
![image](https://user-images.githubusercontent.com/5833609/78464345-385d8580-76b6-11ea-860b-5e598e367e67.png)

**Comment by sachk** [comment]
Yeah, this seems to be on google's end. The process of getting the stream URL from Google will now have to be done clientside, which might not be possible/feasible since it'll require giving the client account credentials.
